backend/src/components/user.js

```diff
@@ -76,9 +76,9 @@ async function getUserInfo(req, res) {
       return res.status(HttpStatus.OK).json(resData)
     }
   } else {
-    //Not an idir user, so just get the guid from the header
-    log.verbose('User Guid is: ', userGuid)
-    userResponse = await getUserProfile(userGuid, null)
+    // BCeID user
```

Comment:
Can we add a comment to explain what's happening here in terms of GUID population in Dynamics?

Reply:
Updated comment.